from __future__ import print_function
import mysql.connector
from mysql.connector import errorcode
cnx = mysql.connector.connect(user='root', password='hello', database='db4660', host='127.0.0.1')
d={
'fname':[
    'Cody',
    'ThuyVi',
    'Shawn',
    'Wyatt',
    'Jamie',
    'Geoff',
    'John',
    'Travis',
    'Laura',
    'Liam',
    'Sam',],
'lname':[
    'Adams',
    'Nguyen',
    'Higa',
    'Trombley',
    'Blue',
    'Chappell',
    'Oliver',
    'Willingham',
    'Bailey',
    'OBrien',
    'Reigel',],
}
import random
for _ in range(100):
    first_part=sd['fname'][random.randint(0,len(sd['fname'])-1)]
    second_part=sd['lname'][random.randint(0,len(sd['lname'])-1)]
    print('{} {}'.format(first_part, second_part))
    cursor.execute("INSERT INTO students (fname,lname,age,addr) VALUES('{}','{}',{},'yes')".format(first_part, second_part,random.randint(0,99)))
    
cursor.execute("SELECT * FROM students")
cursor.execute("DELETE * FROM students")



cursor.close()
cnx.close()
